packages/core/src/challenger/isChallengeSubmitTime.ts

```diff
@@ -0,0 +1,18 @@
+import { MINIMUM_TIME_BETWEEN_ASSERTIONS } from "../constants/index.js";
+import { getLatestChallenge } from "./getLatestChallenge.js";
+import { Challenge } from "./index.js";
+
+export async function isChallengeSubmitTime(): Promise<{isSubmitTime:boolean, currentChallenge: Challenge}> {
+
+	    // Get Last Challenge Data
+		const challengeData = await getLatestChallenge();
+		const currentChallenge = challengeData[1];
```

Comment:
Just fyi you can save a line and make this a little more readable with `[challengeId, currentChallenge] = await getLatestChallenge();`
Let me know what you think :)

Reply:
Done, thank you for the tip!